.github/workflows/e2e-testing.yml

```
@@ -13,12 +14,27 @@ jobs:
+
       - name: Run Playwright tests
-        run: make test-e2e
+        run: pnpm make test-e2e
 
       - uses: actions/upload-artifact@v4
         if: always()
         with:
           name: playwright-report
           path: playwright-report/
-          retention-days: 30
+          retention-days: 30
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)